moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1850,7 +1819,8 @@ bool RobotState::setFromIKSubgroups(const JointModelGroup* jmg, const EigenSTL::
       std::vector<double> ik_sol;
       moveit_msgs::MoveItErrorCodes error;
       const std::vector<double>& climits = consistency_limits.empty() ? std::vector<double>() : consistency_limits[sg];
-      if (solvers[sg]->searchPositionIK(ik_queries[sg], seed, timeout, climits, ik_sol, error))
+      if (solvers[sg]->searchPositionIK(ik_queries[sg], seed, (timeout - elapsed) / sub_groups.size(), climits, ik_sol,
```

Comment:
Shouldn't this be `(timeout - elapsed) / (sub_groups.size() - sg)`?

Reply:
See https://github.com/ros-planning/moveit/pull/1288#discussion_r245912074.